utils/mail.py

```diff
@@ -21,9 +21,39 @@
 from django.conf import settings
 from django.core.mail.message import EmailMessage
 from django.template.loader import render_to_string
-from django.core.mail import get_connection, EmailMultiAlternatives
+from django.core.mail import get_connection
 
 
+def generate_tmp_email(email):
```

Comment:
Could be better but I think it's not too bad and there is a docstring which gives more details.
Unless you have an amazing name proposal I'll keep this one ;)